dsgrid/dimension/time.py

```diff
@@ -291,3 +362,11 @@ def make_time_range(start, end, frequency, leap_day_adjustment):
     elif frequency == datetime.timedelta(days=0):
         return NoOpTimeRange(start, end, frequency, leap_day_adjustment)
     return DatetimeRange(start, end, frequency, leap_day_adjustment)
+
+
+def get_timezone(tz_value: str) -> TimeZone:
+    """convert GEOGRAPHY record time_zone col str to dsgrid TimeZone object"""
```

Comment:
Is geography record relevant here? How is this different from `TimeZone(tz_value)`?